Dockerfile

```diff
@@ -1,24 +1,36 @@
-FROM debian:wheezy
+FROM debian:8.9
+MAINTAINER wekan
```

Comment:
The `MAINTAINER` instruction is [deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated).

Reply:
Should this be ok?

```diff
-MAINTAINER wekan
+LABEL maintainer="wekan"
```

Reply:
@eemeli Updated, let me know if other changes are needed. Thanks!